lib/temporal/client.rb

```diff
@@ -9,15 +9,50 @@
 
 module Temporal
   class Client
+    # @return [String] Namespace used for client calls.
     attr_reader :namespace
 
-    # TODO: More argument to follow for converters, codecs, etc
+    # Create a Temporal client from a service client.
+    #
+    # @param connection [Temporal::Connection] A previously established connection with the
```

Comment:
Hmm... technically this could be a lazy connection.

Reply:
That makes sense, yes, will update